plugins/mysql/mysqlembed.cpp

```diff
@@ -804,6 +804,9 @@ static bool isString(enum_field_types type)
     case MYSQL_TYPE_BLOB:
     case MYSQL_TYPE_STRING:
     case MYSQL_TYPE_VAR_STRING:
+#if MYSQL_VERSION_ID >= 50709
```

Comment:
Perhaps you can say

`#ifdef MRN_HAVE_MYSQL_TYPE_JSON
`
?

Reply:
Considered it, and tested against it, but it seems to be defined outside the client lib.